docs/source/live.rst

```diff
@@ -0,0 +1,172 @@
+
+You might want to disable auto-refresh entirely if your updates are not very frequent, which you can do by setting ``auto_refresh=False`` on the constructor.
+If you disable auto-refresh you will need to call :meth:`~rich.live.Live.refresh` manually or :meth:`~rich.live.Live.update` with ``refresh=True``.
+
+Vertical Overflow
+~~~~~~~~~~~~~~~~~
+
+By default, the live display will display ellipsis if the renderable is too large for the terminal. You can adjust this by setting the
+``vertical_overflow`` argument on the :class:`~rich.live.Live` constructor.
+
+- crop: Show renderable up to the terminal height. The rest is hidden.
+- ellipsis: Similar to crop except last line of the terminal is replaced with "...". This is the default behavior.
+- visible: Will allow the whole renderable to be shown. Note that the display cannot be properly cleared in this mode.
+
+.. note::
+
+    Once the live display stops on a non-transient renderable, the last frame will render as **visible** since it doesn't have to be cleared.
+
+Complex Renders
+~~~~~~~~~~~~~~~
+
+Refer to the :ref:`Render Groups` about combining multiple :class:`RenderableType` together so that it may be passed into the :class:`~rich.live.Live` constructor
+or :meth:`~rich.live.Live.update` method.
+
+For more powerful structuring it is also possible to use nested tables.
+
+
+Print / log
+~~~~~~~~~~~
+
+The Live class will create an internal Console object which you can access via ``live.console``. If you print or log to this console, the output will be displayed *above* the live display. Here's an example::
+
+    import time
+
+    from rich.live import Live
+    from rich.table import Table
+
+    table = Table()
+    table.add_column("Row ID")
+    table.add_column("Description")
+    table.add_column("Level")
+
+    with Live(table, refresh_per_second=4):  # update 4 times a second to feel fluid
+        for row in range(12):
+            live.console.print("Working on row #{row}")
+            time.sleep(0.4)
+            table.add_row(f"{row}", f"description {row}", "[red]ERROR")
+
+
+If you have another Console object you want to use, pass it in to the :class:`~rich.live.Live` constructor. Here's an example::
+
+    from my_project import my_console
+
+    with Live(console=my_console) as live:
+        my_console.print("[bold blue]Starting work!")
+        ...
+
+.. note::
+
+    If you are passing in a file console, the live display only show the last item once the live context is left.
+
+Redirecting stdout / stderr
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To avoid breaking the live display visuals, Rich will redirect ``stdout`` and ``stderr`` so that you can use the builtin ``print`` statement.
+This feature is enabled by default, but you can disable by setting ``redirect_stdout`` or ``redirect_stderr`` to ``False``.
+
+
+Examples
+--------
+
+See `table_movie_live.py <https://github.com/willmcgugan/rich/blob/master/examples/table_movie_live.py>`_ and
```

Comment:
I may end up just replacing replacing table_movie.py with table_movie_live. Since it could serve as a demo of table features and the Live class.

Reply:
Updated the table_movie_live.py -> table_movie.py